todo_app/app.py

```diff
@@ -5,7 +5,7 @@
 app = Flask(__name__)
 app.config.from_object(Config)
 
-# I added the add functionality to the route rather than a seperate page as this seemed more efficient
+# I added add functionality to the route rather than a seperate page as this seemed more efficient
```

Comment:
Personally I'd split these methods out to make it as clear as possible what each route does (even if you're using the same route which is fine). It would also be clearer if we commonise the code involved with rendering the template e.g. using flask's redirect after the successful post

```python
@app.route('/', methods=['GET'])
def index():
	items = get_items()
	return render_template("index.html", items=items)

@app.route('/', methods=['POST'])
def add():
	newItemTitle = request.form['text']
	add_item(newItemTitle)
	return redirect('/') # or return redirect(url_for('index'))
```

Reply:
Thanks for the comments. Im a little behind and need to crack on with the second module project but will revisit this when I can!